localization/locale_zhcn.php

```diff
@@ -1971,9 +1971,7 @@
             "每5秒恢复%d点生命值。",
             "法术穿透提高%d。",
             "使你的盾牌格挡值提高%d。",
-            "Unknown Bonus #%d (%d)",
+            "未知加成 #%d (%d)",
         )
     )
 );
-
-?>
```

Comment:
uhh .. that closing tag is needed.

Reply:
Oh, that was introduced by the auto merge.

..wat?

Reply:
fixed all, thanks